modules/nf-core/deepvariant/makeexamples/main.nf

```diff
@@ -0,0 +1,65 @@
+process DEEPVARIANT_MAKEEXAMPLES {
+    tag "$meta.id"
+    label 'process_high'
+
+    //Conda is not supported at the moment
+    container "nf-core/deepvariant:1.5.0"
+
+    input:
+    tuple val(meta), path(input), path(index), path(intervals)
+    tuple val(meta2), path(fasta)
+    tuple val(meta3), path(fai)
+    tuple val(meta4), path(gzi)
+
+    output:
+    // Output a unique ID (hash) for this task, that can be used to join the gvcf and variant calls later.
+    tuple val("${task.hash}"), val(meta), path("${prefix}.examples.tfrecord-*-of-*.gz"),    emit: examples
+    tuple val("${task.hash}"), val(meta), path("${prefix}.gvcf.tfrecord-*-of-*.gz"),        emit: gvcf
```

Comment:
It would indeed be cleaner to have a unique meta for all individual tasks, and avoid the hash value. I'll change to this approach and try to document it somewhere in the subworkflow information.